mm/slab.c

```diff
@@ -3381,7 +3381,8 @@ static inline void __cache_free(struct kmem_cache *cachep, void *objp,
 	kmemleak_free_recursive(objp, cachep->flags);
 	objp = cache_free_debugcheck(cachep, objp, caller);
 
-	ktsan_memblock_free(objp, cachep->object_size);
+	ktsan_memblock_free(objp, cachep->object_size,
+			!(cachep->flags & SLAB_DESTROY_BY_RCU));
```

Comment:
I don't like that we have this logic in mm/slab.c (not our file). We will need to touch mm/ files when we want to change the condition. Let's do a more robust thing -- pass in cachep object and look at flags inside.